tests/materials/test_materials_service.py

```diff
@@ -55,6 +56,13 @@ def test_create_material_form_creates_admixture():
         assert isinstance(form, AdmixtureForm)
 
 
+def test_create_material_form_creates_custom():
+    with app.test_request_context('/materials/custom'):
+        file, form = MaterialsService().create_material_form('custom')
```

Comment:
There was a typo here which was fixed already. Otherwise, the changes seem fine to me.